modules/components/pages/processors/aws_bedrock_chat.adoc

```diff
@@ -0,0 +1,237 @@
+= aws_bedrock_chat
+= aws_bedrock_chat
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Generates responses to messages in a chat conversation, using the AWS Bedrock API.
+
+Introduced in version 4.34.0.
+
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
+label: ""
+aws_bedrock_chat:
+  model: amazon.titan-text-express-v1 # No default (required)
+  prompt: "" # No default (optional)
+  system_prompt: "" # No default (optional)
+  max_tokens: 0 # No default (optional)
+  stop: 0 # No default (optional)
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All configuration fields, showing default values
+label: ""
+aws_bedrock_chat:
+  region: ""
+  endpoint: ""
+  credentials:
+    profile: ""
+    id: ""
+    secret: ""
+    token: ""
+    from_ec2_role: false
+    role: ""
+    role_external_id: ""
+  model: amazon.titan-text-express-v1 # No default (required)
+  prompt: "" # No default (optional)
+  system_prompt: "" # No default (optional)
+  max_tokens: 0 # No default (optional)
+  stop: 0 # No default (optional)
+  temperature: [] # No default (optional)
+  top_p: 0 # No default (optional)
+```
+
+--
+======
+
+This processor sends prompts to your chosen large language model (LLM) and generates text from the responses, using the AWS Bedrock API.
+
+For more information, see the https://docs.aws.amazon.com/bedrock/latest/userguide[AWS Bedrock documentation^].
+
+== Fields
+
+=== `model`
+
+The model ID to use. For a full list, see the https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html[AWS Bedrock documentation^].
```

Comment:
@rockwotj. I have made minor amends to the field descriptions. Please could you check that all is correct?

Reply:
Looks good! I will update the original source with these.